src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents an association between a Person and a Tag in the address book.
+ */
+public class Tagging {
+	private Person person;
+	private Tag tag;
+	private boolean isAddition;
+	
+	public Tagging(Person person, Tag tag, boolean isAddition) {
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.